### PR TITLE
Add x-forward options to projects - AB#7604

### PR DESCRIPTION
**Pull request by sslaws**

### CODE REVIEW

File: Apps/Common/src/AspNetConfiguration/StartupConfiguration.cs [hunk 1]
```diff
@@ -205,6 +206,32 @@ public void UseAuth(IApplicationBuilder app)
             app.UseAuthentication();
         }
 
+        /// <summary>
+        /// Configures the app to use x-forwarded-for headers to obtain the real client IP.
+        /// </summary>
+        /// <param name="app">The application builder provider.</param>
+        public void useForwardHeaders(IApplicationBuilder app)
```
Comment (FuriousLlama) [inline Apps/Common/src/AspNetConfiguration/StartupConfiguration.cs]: nit: uppercase first letter

File: Apps/Immunization/src/appsettings.Development.json [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
-    "OpenIdConnect": {
-        "Audience": "healthgateway-local",
-        "Authority": "https://sso-dev.pathfinder.gov.bc.ca/auth/realms/ff09qn3f",
-        "ClientId": "healthgateway-local",
-        "RequireHttpsMetadata": "false"
-    },
-    "AllowOrigins": "http://localhost:5000",
-    "Environment": {
-        "EnableDebug": true
+  "Logging": {
```
Comment (FuriousLlama) [inline Apps/Immunization/src/appsettings.Development.json]: indentation change?

**Comment by healthgatewaybc** [comment]
Common Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=commonlibrary_pr_188)

**Comment by healthgatewaybc** [comment]
Patient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=patientservice_pr_188)

**Comment by healthgatewaybc** [comment]
Immunization Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=immunization_pr_188)

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_188)

File: Apps/Common/src/AspNetConfiguration/StartupConfiguration.cs (hunk 1 above)
Comment (sslaws) [reply Apps/Common/src/AspNetConfiguration/StartupConfiguration.cs]: Fixed

File: Apps/Immunization/src/appsettings.Development.json (hunk 2 above)
Comment (sslaws) [reply Apps/Immunization/src/appsettings.Development.json]: Yes, apparently indent was off.

**Comment by healthgatewaybc** [comment]
Common Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=commonlibrary_pr_188)

**Comment by healthgatewaybc** [comment]
Patient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=patientservice_pr_188)

**Comment by healthgatewaybc** [comment]
Immunization Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=immunization_pr_188)

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_188)

**Comment by healthgatewaybc** [comment]
WebClient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=webclient_pr_188)

**Comment by healthgatewaybc** [comment]
Common Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=commonlibrary_pr_188)

**Comment by healthgatewaybc** [comment]
Immunization Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=immunization_pr_188)

**Comment by healthgatewaybc** [comment]
Patient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=patientservice_pr_188)

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_188)

**Comment by healthgatewaybc** [comment]
WebClient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=webclient_pr_188)

**Comment by healthgatewaybc** [comment]
Patient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=patientservice_pr_188)

**Comment by healthgatewaybc** [comment]
Common Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=commonlibrary_pr_188)

**Comment by healthgatewaybc** [comment]
WebClient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=webclient_pr_188)

**Pull request merged**